### PR TITLE
Ticket6

### PR DESCRIPTION
**Pull request by yos244**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const app = require("../app.js");
 const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data");
 const db = require("../db/connection.js");
+const { response } = require("../app.js");
```
Comment (Geo-Beard) [inline __tests__/app.test.js:6]: Remove this - I know it's an automatic thing - but it's crept in again :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -49,6 +50,58 @@ describe(" api/categories", () => {
...
+          created_at: new Date(1610964101251).toString(),
+        });
+      });
+  });
+  test("GET: 200 /api/reviews/:review_id/comments", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual([
```
Comment (Geo-Beard) [inline __tests__/app.test.js:77]: Add an expect in here that checks the order of the comments coming back - most recent comments first - jest-sorted needs to be used.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -49,6 +50,58 @@ describe(" api/categories", () => {
...
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          votes: 5,
+          category: `dexterity`,
+          owner: `philippaclaire9`,
+          created_at: new Date(1610964101251).toString(),
+        });
+      });
+  });
+  test("GET: 200 /api/reviews/:review_id/comments", () => {
```
Comment (Geo-Beard) [inline __tests__/app.test.js:72]: Add another test for a valid review ID that has no comments - should return an empty array.

**Review by Geo-Beard — COMMENTED** [review]
Looks good overall - couple of comments that need implementing - once you've implemented those feel free to merge and move on :)
